src/AppMenuSearch.cc

```
@@ -0,0 +1,408 @@
+        if (action && action->property("isAppMenuSearchProxy").toBool() == true) {
+            m_searchMenu->removeAction(action);
+            // Detach action from its group before scheduling deletion
+            if (QActionGroup *group = action->actionGroup()) {
+                group->removeAction(action);
+            }
+            action->deleteLater();
+        }
+    }
+}
+
+void AppMenuSearch::invalidateCandidates()
```

Cppcheck (2.21.0)

[style] The function 'invalidateCandidates' is never used.

(unusedFunction)